src/lib/pages/instantiate/instantiate.tsx

```diff
@@ -234,6 +237,7 @@ const Instantiate = ({ onComplete }: InstantiatePageProps) => {
   useEffect(() => {
     setValue("codeHash", "");
     setTab(MessageTabs.JSON_INPUT);
+    setValue(`msgInput.${yourSchemaInputFormKey}`, "{}");
```

Comment:
Should we reset `msgInput.${jsonInputFormKey}`?

Reply:
From what I've tried, using reset is a bit overcomplicating things here since we are only targeting two fields (codeHash and schema input) and one of which is nested (resulting in less-clean code)